osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -48,29 +50,21 @@ public class DrawableJudgement : CompositeDrawable
         /// <param name="result">The judgement to visualise.</param>
         /// <param name="judgedObject">The object which was judged.</param>
         public DrawableJudgement(JudgementResult result, DrawableHitObject judgedObject)
+            : this()
         {
-            Result = result;
-            JudgedObject = judgedObject;
+            Apply(result, judgedObject);
+        }
 
+        public DrawableJudgement()
+        {
             Size = new Vector2(judgement_size);
+            Origin = Anchor.Centre;
```

Comment:
Is this required? Seems to break taiko judgement positioning (and maybe mania, depends on how it positions it).

Can it be moved to osu! instead if it's required?

Reply:
Interesting, I didn't even notice. I just moved it here because I thought it was shared across ruleset, it can be reverted without issue.

Reply:
Actually doesn't seem to be as a result of this PR and centre origin seems to be the general use case (all our rulesets + tau + sentakki)